pkgs/by-name/so/solhint/package.nix

```
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+}:
+
+buildNpmPackage {
+  pname = "solhint";
+  version = "v6.0.0";
```

```suggestion
  version = "6.0.0";
```
Version must start with a digit